.github/workflows/main.yml

```diff
@@ -2,8 +2,8 @@ on: [pull_request]
 jobs:
   build:
     env:
-      YOCTO_VERSION: 3.2.3
-      YOCTO_BRANCH: gatesgarth
+      YOCTO_VERSION: 3.3.2
+      YOCTO_BRANCH: hardknott
```

Comment:
As we discussed, I think we want meta-rust to support dunfell for the foreseeable future.  Therefore we should probably configure the CI to use dunfell

Reply:
Then there should be 2 or 3 CI jobs to test against dunfell, hardknott and honister (which is what this layer claims compatibility with) only the currently used gatesgarth is the least important one as gatesgarth is EOL.

Reply:
I don't disagree with that, but dunfell is what I want to see pass before merging this PR :smile:

Reply:
hardknott with 1.50 bitbake is affected the same as dunfell with 1.46 and both got the same patch in bitbake, so if hardknott is passing, then dunfell will as well (if it wasn't already failing before this PR).

Reply:
I tend to agree with @srwalter here, and I'd support `dunfell` to be included in the test matrix. Besides `dunfell`, `master` would be great too. The other releases we can assume to be good if those two are good.

Reply:
I've triggered CI for dunfell, hardknott and master with this PR and all 3 were canceled after 5 hours during testing SDK tasks, but you can check the logs that it parsed and built something.

Reply:
And the last run with hardknott again failed due to disk space:
```
WARNING: The free space of /home/srwalter/actions-runner/_work/meta-rust/meta-rust/poky/build/tmp (/dev/mapper/ubuntu--vg-ubuntu--lv) is running low (0.097GB left)
WARNING: The free space of /home/srwalter/actions-runner/_work/meta-rust/meta-rust/poky/build/downloads (/dev/mapper/ubuntu--vg-ubuntu--lv) is running low (0.097GB left)
WARNING: The free space of /home/srwalter/actions-runner/_work/meta-rust/meta-rust/poky/build/sstate-cache (/dev/mapper/ubuntu--vg-ubuntu--lv) is running low (0.097GB left)
```